chat-plugins/info.js

```diff
@@ -1337,6 +1337,18 @@ exports.commands = {
 		if (showAll || target === 'tiering' || target === 'tiers' || target === 'tier') {
 			buffer.push("<a href=\"https://www.smogon.com/ingame/battle/tiering-faq\">Tiering FAQ</a>");
 		}
+		if (showAll || toId(target).startsWith('lostpass') || target === 'pwreset') {
+			buffer.push("PM a member of Global Staff (@ in the Help room) to facilitate password resets.");
+		}
+		if (showAll || toId(target) === 'lostteams' || toId(target) === 'deletedteams') {
+			buffer.push("Have you cleared your cookies? This deletes your teams. You may  have done this accidentally when clearing your history, or by using  private browsing. Always back up your teams to a pastebin!");
+		}
+		if (showAll || target === 'ladder') {
+			buffer.push("Ladder battles are any matches you start with the 'Battle!' button besides unrated random battles.");
```

Comment:
This is really obvious information and not what you'd expect from a "ladder faq"